freeipa/src/main/java/com/sequenceiq/freeipa/flow/stack/upgrade/ccm/UpgradeCcmService.java

```diff
@@ -90,6 +106,27 @@ public void checkPrerequsities(Long stackId) {
         }
     }
 
+    private void checkCcmV2ServerConnectivity(Long stackId) {
+        Map<String, String> result = upgradeCcmOrchestratorService.checkCcmV2Connectivity(stackId, ccmV2ServersCidr);
+        LOGGER.debug("Returned result from CCMv2 Connectivity Check: {}", result);
+        if (result != null) {
+            String failures = result.values().stream()
+                    .map(resultJson -> JsonUtil.readValueOpt(resultJson, ConnectivityCheckResponse.class))
```

Comment:
maybe we should log if there is an issue here. `JsonUtil.readValueOp` doesn't log if there is an IOException

Reply:
Logged inside the stream